vhdbuilder/packer/install-dependencies.sh

```diff
@@ -351,15 +351,7 @@ fi
 MULTI_ARCH_CNI_PLUGIN_VERSIONS="
 0.9.1
 "
-
-if [[ $(isARM64) == 1 ]]; then
-  ARM64_CNI_PLUGIN_VERSIONS_IN_USE="
-  0.8.7
```

Comment:
Now on AKS-RP side, kubenet CNI 0.9.1 is used for both amd64/arm64 for kubenet plugin scenario. so this special 0.8.7 installation is not needed anymore.